README.md

```diff
@@ -38,7 +40,120 @@ class GuiceModule extends AbstractModule {
 By using Gluon Ignite, you not only get dependency injection inside your application class, but also 
 within your FXML controllers too. Even though the sample above shows a Guice context, as mentioned 
 Gluon Ignite also supports other DI frameworks. By using a DaggerContext or SpringContext in your application, 
-it can be easily set up to work with with those frameworks instead.
+it can be easily set up to work with those frameworks instead.
+
+### Micronaut support
+
+Ignite implementation for Micronaut differs from the other implementations, and has much deeper integration with Micronaut framework.
+
+#### _Application main class_
+
+Ignite Micronaut provides a special implementation of JavaFX `Application` class, 
+which has to be configured as your main class: `com.gluonhq.ignite.micronaut.FXApplication`. 
```

Comment:
What if you already have a class that extends MobileApplication?

Reply:
Tough... in DI you'd like to be everything "injectable", including Application itself. 
So far,  I could not find a way to do it with Micronaut. Hopefully it is possible.

Reply:
This should be resolved with commit cae5b6c